resource-aggregate/pb/commands.proto

```diff
@@ -56,30 +51,28 @@ message ResourceId {
 
 // https://github.com/openconnectivityfoundation/cloud-services/blob/master/swagger2.0/oic.wk.rd.swagger.json#L35
 message PublishResourceLinksRequest {
-    AuthorizationContext authorization_context = 1;
-    repeated Resource resources = 2;
-    string device_id = 3;
-    CommandMetadata command_metadata = 4;
+    repeated Resource resources = 1;
+    string device_id = 2;
+    CommandMetadata command_metadata = 100;
 }
 
 message PublishResourceLinksResponse {
-    AuditContext audit_context = 1;
-    repeated Resource published_resources = 2;
-    string device_id = 3;
+    repeated Resource published_resources = 1;
+    string device_id = 2;
+    AuditContext audit_context = 100;
```

Comment:
Why 100? Don't like this numbering.

Reply:
I want to add space for new properties,

Reply:
What prevents you to to add number 4 to new property, so index of a new property will be bigger than audit_context?